include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
How are you linking your application? You have to link the libraries Ben pointed out.